examples/plugins/afk.js

```diff
@@ -0,0 +1,50 @@
+// Plugins are loaded using the bot.loadPlugin() function
```

Comment:
this information should be and is in api.md, theres no reason to rewrite a description of bot.loadplugin

Reply:
It is hard to explain how to make a plugin without a decent understanding of how plugins are loaded.

Reply:
you should just link to the api.md entry for bot.loadplugin then